VERSION

```diff
@@ -1 +1 @@
-1.0.1
+2.0.0
```

Comment:
I went with a major version bump due to the incompatible change from the `token` kwarg to `auth_header` in a couple of functions (even though those are lower level).